roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -0,0 +1,105 @@
+---
+  - fail: msg="This role requires the following vars to be defined. openshift_hosted_logging_master_public_url, openshift_hosted_logging_hostname, openshift_hosted_logging_elasticsearch_cluster_size"
+    when: "openshift_hosted_logging_hostname is not defined or
+          openshift_hosted_logging_elasticsearch_cluster_size is not defined or
+          openshift_hosted_logging_master_public_url is not defined"
+
+  - name: Create temp directory for kubeconfig
+    command: mktemp -d /tmp/openshift-ansible-XXXXXX
+    register: mktemp
+    changed_when: False
+
+  - name: Copy the admin client config(s)
+    command: >
+      cp {{ openshift_master_config_dir }}/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
+    changed_when: False
+
+  - name: "Create logging project"
+    command: oadm new-project logging
```

Comment:
@abutcher Thanks. Good catch. All the `oc` and `oadm` commands should use `--config={{ mktemp.stdout }}/admin.kubeconfig` correct?